nuxt.config.js

```diff
@@ -11,7 +11,9 @@ module.exports = {
       { hid: 'description', name: 'description', content: 'Nuxt.js project' }
     ],
     link: [
-      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
+      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
+      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Open+Sans:300,400&amp;subset=latin-ext' },
+      { rel: 'stylesheet', href: '/fonts/flaticon/flaticon.css' }
```

Comment:
We got a section with css assets, please move there import of flaticon

Reply:
Good point, this will be a better place for flaticon import :)